README.md

```diff
@@ -34,29 +34,9 @@ bundle exec rake contacts:import_hmrc DATA_FILE=db/contact-records.csv
 bundle exec rspec
 ```
 
-## Redirecting removed contacts
+### Further documentation
 
-The Web UI of the app issues a “gone” item to the publishing api when a contact is deleted. Sometimes the correct thing to do is to issue a “redirect” instead. There is no web UI for this but there are two rake tasks to achieve this depending on the state of the contact to be redirected.
```

Comment:
Oh, this is completely different as of #838 - contacts are ALL redirected, rather than `gone`. We can probably just delete this section!

...and, come to think of it, some of the rake tasks 🤔 

I'll raise a PR to do that - should have done it in #838 - and then we can rebase this PR.

Reply:
PR: #846. 👀 🙏